starcity-properties/move-out-app
# Implement different application flows

- Prototype of all the application flows

### Testing
- Pull the branch
- Install packages with `yarn`
- Run with `react-native run-ios`
- Test the different screen flows with guide from the Figma mockups. It'll make sense if you play the mockups to ensure the flows match the mock ups flow.
